code/include/env/mcause.h

```diff
@@ -4,7 +4,38 @@
 #define MCAUSE_EXCEPTION_SHIFT 0
 #define MCAUSE_INTERRUPT_SHIFT 31
```

Comment:
Add a description here, using comments to explain MCAUSE is a register of fixed length (32 bits).